packages/fast-usdc/src/distribute-fees.core.js

```diff
@@ -0,0 +1,92 @@
+/** @file core eval module to collect fees. */
+import { AmountMath } from '@agoric/ertp';
+import { floorMultiplyBy } from '@agoric/zoe/src/contractSupport/index.js';
+import { E } from '@endo/far';
+import { fromExternalConfig } from './utils/config-marshal.js';
+
+/**
+ * @import {DepositFacet} from '@agoric/ertp';
+ * @import {FastUSDCCorePowers} from '@agoric/fast-usdc/src/fast-usdc.start.js';
+ * @import {CopyRecord} from '@endo/pass-style'
+ * @import {BootstrapManifestPermit} from '@agoric/vats/src/core/lib-boot.js'
+ * @import {LegibleCapData} from './utils/config-marshal.js'
+ */
+
+/**
+ * @typedef {{ destinationAddress: string } &
+ *  ({ feePortion: Ratio} | {fixedFees: Amount<'nat'>}) &
+ *  CopyRecord
+ * } FeeDistributionTerms
+ */
+
+const kwUSDC = 'USDC'; // keyword in AmountKeywordRecord
+const issUSDC = 'USDC'; // issuer name
+
+const trace = (...args) => console.log('FUCF', ...args);
```

Comment:
does importing `makeTracer` create problems? this is fine, just curious

Reply:
no, I didn't run into problems.

This is left over from an earlier phase where I was avoiding all the builder stuff, since the core-eval is so simple. Then I realized the a3p core eval stuff expects it, so I stopped avoiding it.